Example/Backpack/Views/HorizontalNavigationItemWithBackground.swift

```diff
@@ -0,0 +1,104 @@
+//
+/*
```

Comment:
why is this implemented in swift and obj-c?

Reply:
It's not? The item(this file) is in Swift and the option(`BPKHorizontalNavigationOptionWithBackground`) is in Objective-C